common/src/main/java/org/opensearch/ml/common/transport/model/MLUpdateModelRequest.java

```diff
@@ -45,8 +46,25 @@ public MLUpdateModelRequest(StreamInput in) throws IOException {
     @Override
     public ActionRequestValidationException validate() {
         ActionRequestValidationException exception = null;
+
         if (updateModelInput == null) {
-            exception = addValidationError("Update Model Input can't be null", exception);
+            return addValidationError("Update Model Input can't be null", exception);
+        }
+
+        String modelName = updateModelInput.getName();
+        String description = updateModelInput.getDescription();
+
+        if (modelName != null && !isSafeText(modelName)) {
```

Comment:
Similar checking in multiple places ? Can we build some common util method ?

Reply:
Yeah, I thought about that and felt too lazy to refactor 😄. But I fixed it in the second revision. Thanks for insisting on the highest standards!